_i18n/de/resources/user-guides/securely_purchase.md

```diff
@@ -1,43 +1,43 @@
 
-An incomplete list of Exchanges that support Monero is on our [Merchants page]({{ site.baseurl }}/community/merchants/#exchanges).
+Eine unvollständige Liste von Börsen, die Monero unterstützen, findet sich auf unserer [Händlerseite]({{ site.baseurl }}/community/merchants/#exchanges).
 
-## Step 2: Download and create a Paper Wallet on a secure and air-gapped computer.
-Download the @paper-wallet generator at: [moneroaddress.org](https://moneroaddress.org) and copy it to a USB stick (Direct link: [https://github.com/moneromooo-monero/monero-wallet-generator/archive/master.zip](https://github.com/moneromooo-monero/monero-wallet-generator/archive/master.zip)).
+## Schritt 2: Ein Paper-Wallet auf einem sicheren, mittels Airgap isolierten Computer herunterladen und erstellen
+Lade einen Generator für @Paper-Wallets auf [moneroaddress.org](https://moneroaddress.org) herunter und kopiere ihn auf einen USB-Stick (direkter Link: [https://github.com/moneromooo-monero/monero-wallet-generator/archive/master.zip](https://github.com/moneromooo-monero/monero-wallet-generator/archive/master.zip)).
 
-Unzip and open the paper wallet generator (monero-wallet-generator.html) into a web browser on an air-gapped (@airgap) computer that hasn't been used before, or has had a clean installation of the operative system.
+Entpacke den Paper-Wallet-Generator (monero-wallet-generator.html) und öffne ihn in einem Webbrowser auf einem mittels @Airgap isolierten Computer, der zuvor nicht verwendet worden ist oder auf welchem das Betriebssystem sauber neuinstalliert wurde.
 
-Your paper wallet will have four important items:
+Dein Paper-Wallet hat vier wichtige Dinge:
 
-- Monero Public @Address: The public address is used to receive funds to the @wallet. You give this to anyone who will be sending funds to your wallet.
+- Eine öffentliche Monero-@Adresse: Die öffentliche Adresse wird zum Empfangen von Geldern im @Wallet genutzt. Du gibst diese Adresse jedem, der Gelder auf dein Wallet senden wird.
 
-- Monero @Mnemonic-Seed: The mnemonic seed is a method of storing the entire wallet that is easily recognizable to humans.  This is all you need to restore your wallet at a later date.
+- Moneros @mnemonischer-Seed: Der mnemonische Seed ist eine für den Menschen klar zu erkennende Methode, mit welcher du dein Wallet sicherst. Mit diesem Seed hast du alles, was du brauchst, um dein Wallet zu einem späteren Zeitpunkt wiederherzustellen.
```

Comment:
Removed "für den Menschen" since it sounds weird (at least in German). I'll leave it as "method" for now though; I get your point, but I couldn't make up a better way to phrase it.